tsl/test/sql/data_node.sql

```diff
@@ -656,9 +656,26 @@ SELECT * FROM add_data_node('data_node_1', 'localhost', database => 'data_node_1
 SELECT * FROM add_data_node('data_node_99', host => 'localhost');
 \set ON_ERROR_STOP 1
 
+
+-- Test adding bootstrapped data node where extension owner is different from user adding a data node
+DROP DATABASE IF EXISTS data_node_6;
+SET ROLE :ROLE_CLUSTER_SUPERUSER;
+CREATE DATABASE data_node_6;
+\c data_node_6
+SET client_min_messages = ERROR;
+-- Creating an extension as superuser
+CREATE EXTENSION timescaledb;
+RESET client_min_messages;
+
+\c :TEST_DBNAME :ROLE_CLUSTER_SUPERUSER;
+SET ROLE :ROLE_1;
```

Comment:
```suggestion
\c :TEST_DBNAME :ROLE_1;
```

Reply:
Actually this does not work. We need to connect to test db as a super user in order to drop databases later on. So either we keep it as it is or we need to connect as superuser before dropping the databases.